src/styles/shared/components/Buttons/_index.scss

```diff
@@ -0,0 +1,166 @@
+@use "sass:map";
+@use "@hypothesis/frontend-shared/styles/mixins/focus";
+@use './mixins';
+@use "../../variables" as var;
+
+// Design tokens from variables
```

Comment:
The use of design tokens here and the `configure` mixin pattern is new for us, but I think quite potentially useful. I'll explain more clearly when I've had a weekend of rest, but a bit more info here https://sass-lang.com/documentation/at-rules/use#with-mixins

Reply:
This `$-settings` map maps relevant variables (these variables belong to the shared package's SASS) to "tokens" that have meaning to this module. This allows this module to have the same _default_ configuration wherever it is used, and provides a layer of design abstraction between variable naming and their context in this module.

The `@configure` mixin allows targeted configuration/override of any or all of the `$-settings` wherever this module is `@use`d. The usage pattern, then, would be:

```
@use `@hypothesis/frontend-shared/styles/components/Buttons` as buttons;
// OPTIONALLY CONFIGURE
@include buttons.configure($settings: ('border-radius': 5px));
@include buttons.styles;
```